controller/src/main/java/org/jboss/as/controller/parsing/ExtensionXml.java

```diff
@@ -72,7 +72,12 @@ public void writeExtensions(final XMLExtendedStreamWriter writer, final ModelNod
         }
     }
 
-    public void parseExtensions(final XMLExtendedStreamReader reader, final ModelNode address, final Namespace expectedNs, final List<ModelNode> list)
+    @Deprecated
```

Comment:
All the `@Deprecated` should use the "since" argument

Reply:
This is almost done, and CI looks good. We can add the since argument later. Since this is blocking some other work in WildFly full, let's move on with it now so it can be included in today's release.

Reply:
@yersan FYI these deprecated methods will be gone fairly shortly, I just needed them to hang around long enough to not break WildFly.

Reply:
Ok, I won't create any follow-up PR then, thanks!